exercises/APE8/APE08-E3.tex

```diff
@@ -0,0 +1,76 @@
+\subsection*{a.}
+To show that this game allows a correlated equilibrium for which each player receives a payoff higher than 4. We can see that if we take same correlated strategies for all the payoff non zero, we have a payoff higher than 4. So we have :
+$$ \mu(c_1,c_2) = \left\{ 
+\begin{array}{l l}
+    0 & \mbox{if} \ (c_1,c_2) \in \left\{(x_1,x_2),(y_1,y_2),(z_1,z_2)\right\}\\                      
+     \frac{1}{6} & \mbox{otherwise} 
+\end{array} \right.$$
+
+We take $\frac{1}{6}$ because the sum of the correlated strategies equal to one. After that we have to check if it's a correlated equilibrium. Recall that a correlated equilibrium for a game in strategic form must satisfy the following strategic incentive constraints :
+\begin{align*}
+			\sum_{c_{-i} \in C_{-i}} \mu(c) \, \big( u_i(c_{-i}, \; c_i) - u_i(c_{-i}, \; e_i) \big) \geq 0, \quad \forall i \in N, \ \forall c_i \in C_i, \ \forall e_i \in C_i,
+		\end{align*}
+		where $c = (c_{-i}, c_i)$, $c_{-i}$ being the strategy of the players other than $i$. These constraints require that the players cannot improve their payoff by deviating from the correlated strategy $\mu$ suggested by the mediator.
+
+
+To do less computation, we can notice that the strategic form is symmetric and so the constraints are the same for all actions that the mediator could propose to player 1 or 2. Therefore we can just check the case for example where the mediator proposes to the player 2 to play $x_2$ :
+
+- $i=2$ $c_2=x_2$  $e_2=y_2$
+\begin{equation*}  
+\frac{1}{6}(5-0) + \frac{1}{6}(4-5) = \frac{2}{3} \ge 0 
+\end{equation*}
+- $i=2$ $c_2=x_2$  $e_2=z_2$
+\begin{equation*} 
+\frac{1}{6}(5-4) + \frac{1}{6}(4-0) = \frac{5}{6} \ge 0 
+\end{equation*}
+This means that the players 2 has no interest in playing $y_2$ or $z_2$ rather than $x_2$. And so it's a correlated equilibrium and the payoff equal to $\frac{27}{6} = 4.5 \ge 4$
+
+\subsection*{b.}
+
+We can directly see that to maximize the payoff of the player 2, the correlated strategies $\mu_2(x_1,x_2)$ , $\mu_2(y_1,y_2)$ and $\mu_2(z_1,z_2)$ are zero because the associated payoff is zero. So we have to look the six other correlated strategies. Before checking strategic incentive constraints, we can also notice that the correlated strategy with the same payoff for the players 2 will be equals because we have a symmetric game. We write :
```

Comment:
The justification for setting $\mu_2(x_1,x_2) = \mu_2(y_1,y_2) = \mu_2(z_1,z_2) = 0$ is not convincing. Even if their coefficients is zero in the objective, setting a nonzero probability might allow to take higher values for the other probabilities.